src/sentry/conf/server.py

```diff
@@ -301,6 +301,7 @@
     'social_auth.backends.github.GithubBackend',
     'social_auth.backends.bitbucket.BitbucketBackend',
     'social_auth.backends.trello.TrelloBackend',
+    'social_auth.backends.asana.AsanaBackend',
```

Comment:
@mattrobenolt it is required. talked to cramer about it in slack, but asana won't work unless it's included here